sklearn/covariance/tests/test_robust_covariance.py

```diff
@@ -34,6 +34,9 @@ def test_mcd(global_random_seed):
     # 1D data set
     launch_mcd_on_dataset(500, 1, 100, 0.02, 0.02, 350, global_random_seed)
 
+    # n_samples == n_features
+    launch_mcd_on_dataset(20, 20, 0, 0.1, 0.1, 50, global_random_seed)
```

Comment:
I think the CI test is failing because `tol_support = 50` in the `launch_mcd_on_dataset` function. We should always have `tol_support <= n_samples` here.